app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
- [ ] Great job so far but I can't see any views for users even after I added some data to your seeds and run it through rails c so please fix the issue shown here and kindly add some data to your seeds so just anyone can run rails db:seed and be able to see all your views since for now I couldn't see them

![image](https://user-images.githubusercontent.com/58553711/223726857-4bc9d5ec-3cbe-4049-8745-09f60d166bc0.png)